.github/ISSUE_TEMPLATE/issue_template.yml

```
@@ -64,6 +64,17 @@ body:
     validations:
       required: false
 
+  - type: markdown
+    attributes:
+      value: '**Accessibility:**'
+  - type: textarea
+    id: notes
```

Let's change this `id` value to `accessibility` so it stays unique to the section and the form works properly when submitted. 🙂 

```suggestion
    id: accessibility
```